packages/webcomponents/src/components/checkout/checkout.tsx

```diff
@@ -9,6 +9,7 @@ import { checkPkgVersion } from '../../utils/check-pkg-version';
 
 @Component({
   tag: 'justifi-checkout',
+  shadow: true,
```

Comment:
I think we should make it so that `StyledHost` has shadowDom so we dont have to worry about being consistent about where we put this